backend/app/api/endpoints/openapi_responses.py

```
@@ -145,7 +148,591 @@ def _task_to_response_object(
+    # enable_deep_thinking is extracted from tools parameter
+    enable_deep_thinking = tool_settings.get("enable_deep_thinking", False)
+    try:
+        chat_config = config_builder.build(
+            override_model_name=model_info.get("model_id"),
+            force_override=model_info.get("model_id") is not None,
+            enable_clarification=False,  # Not supported in OpenAPI yet
+            enable_deep_thinking=enable_deep_thinking,
+            task_id=task_id or 0,
+        )
+    except ValueError as e:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)